src/algo/camera-age/l500/camera-age.h

```diff
@@ -14,16 +14,16 @@ namespace l500 {
 
 class work_week
 {
-    unsigned man_year;
-    unsigned man_ww;
+    unsigned year;
+    unsigned ww;
 
 public:
     work_week( unsigned man_year, unsigned man_ww )
-        : man_year( man_year )
-        , man_ww( man_ww )
+        : year( man_year )
+        , ww( man_ww )
     {
     }
-    work_week(std::time_t& time);
+    work_week( const std::time_t& time );
 
```

Comment:
How about a way to create from today?
```
static work_week from_today();
```

Reply:
And, below:
As we discussed, no need for `get_work_weeks_since`, right? It will be `work_week::from_today() - l500::get_manufacture_work_week( serial_number )`?